Source/ResponseSerialization.swift

```diff
@@ -514,6 +569,9 @@ public final class StringResponseSerializer: ResponseSerializer {
 
         data = try dataPreprocessor.preprocess(data)
 
+        #if os(Linux)
+        let actualEncoding = String.Encoding.isoLatin1
```

Comment:
Here we’ll need a Linux equivalent to `CFStringConvertIANACharSetNameToEncoding`.

Reply:
I found this: https://opensource.apple.com/source/CF/CF-476.18/CFStringUtilities.c.auto.html

Reply:
I don't think we should import any outside code to support this on Linux. Instead, we could try to create a more Swift native cross platform solution.

Reply:
Added a new initializer for `String.Encoding` that takes an IANA charset name. Right now it supports the same names as the code it replaces, but we could also add missing ones from [the list](https://www.iana.org/assignments/character-sets/character-sets.xhtml).